packages/deb/hazelcast/DEBIAN/control

```diff
@@ -1,6 +1,6 @@
 Package: hazelcast
 Source: hazelcast
-Version: 4.2020.10
+Version: 5.0-SNAPSHOT
```

Comment:
I think the version should be something like `5.2021.8` (8 for August release). Don't ask me why :-)

Reply:
Hmm, thanks. I am not sure, but this change should be made after the full release probably. The reason I went with snapshot is to pass integration tests and thus, make adjustments according to Hazelcast 5.0 changes.